CHANGELOG.md

```diff
@@ -4,6 +4,26 @@
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v0.13.3...master)
 
+## General
+
+### What's New
+
+- In most cases, opaque integer handles are now used to pass data over the FFI.
+  This should be more robust, and allow detection of many types of errors that would
+  previously cause silent memory corruption.
+
+  This should be mostly transparent, but is a semi-breaking semantic change in the
+  case that something throws an exception indicating that the rust code paniced (which
+  should only occur due to bugs anyway). If this occurs, all subsequent operations
```

Comment:
You mentioned in https://github.com/mozilla/application-services/pull/335#issuecomment-436088075 that it's not our place as a library to abort on panic, but I wonder if we should offer guidance that getting a panic means the app should terminate immediately, instead of trying to recover.

Reply:
That's already what everybody does, AFAICT. But it might be worth talking about in case that changes.